### ISSUE TITLE
How to convert my GEOJson data to Shape file and save it in SDCard?

### ISSUE THREAD

**Issue by klower** [Esri/arcgis-runtime-samples-android#427]
How to convert my GEOJson data to Shape file and save it in SDCard?

**Comment by TADraeseke** [comment]
Support for shapefiles is coming to the ArcGIS Runtime SDK very soon. Stay tuned!

**Comment by TADraeseke** [comment]
Hi @klower 

Support for shapefiles has been added to our 100.2 [release of the ArcGIS Runtime SDK for Android](https://blogs.esri.com/esri/arcgis/2017/12/18/whats-new-in-arcgis-runtime-sdk-100-2-0/).
 
Here's our first sample using shapefiles: https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/symbolize-shapefile

Let us know if you have any more questions either here or on the [GeoNet community](https://community.esri.com/community/developers/native-app-developers/arcgis-runtime-sdk-for-android/content?filterID=contentstatus%5Bpublished%5D~objecttype~objecttype%5Bthread%5D).

**Comment by khrlimam** [comment]
@TADraeseke is release 100.2 support shapefiles also means exporiting geojson to shapefile?

**Comment by mbcoder** [comment]
@klower we don't directly support geoJSON, but it isn't hard to parse the JSON and extract the geometry which is basically a set of points.  From this you would be able to build geometries using the API to generate your points lines or polygons.

Once you've extracted the geometry from the GeoJSON you can store it in graphics overlays (temporary data), feature services or even shape files.

So your use-case is achievable using the current release (shapefiles are supported)

To make life easier will aim to add direct support for GeoJSON in a future release.

Note that currently we have support from reading and writing to existing shapefiles, but not creating new shapefiles.  Creation of new shapefiles will be added in a future release.

**Comment by khrlimam** [comment]
@mbcoder I see Thanks.
